packages/p2p/src/components/orders/order-table-row.jsx

```diff
@@ -62,51 +69,138 @@ const OrderRowComponent = observer(({ data: order, style }) => {
 
     const offer_amount = `${amount_display} ${account_currency}`;
     const transaction_amount = `${price_display} ${local_currency}`;
+    const is_buy_order_type_for_user = (is_buy_order && !is_my_ad) || (is_sell_order && is_my_ad);
+    const order_type = is_buy_order_type_for_user ? localize('Buy') : localize('Sell');
 
+    const title = () => {
+        const send_amount = is_buy_order_type_for_user ? transaction_amount : offer_amount;
+        return (
+            <React.Fragment>
+                <div className='orders__expander-content__title' as='p'>
+                    <Text size='sm' weight='bold'>
```

Comment:
Nit, I think it's a good idea to always pass `line_height` so as to be more explicit.